src/utils/event-manager.ts

```diff
@@ -47,6 +51,11 @@ export enum DetailType {
    */
   PROVISION_FAILURE = 'provisionFailure',
 
+  /**
+   * Event detail type for provisioning request.
+   */
+  DEPROVISION_REQUEST = 'deprovisionRequest',
+  /**
```

Comment:
i think this line is not needed.

Reply:
You're right, I removed the extra comment block. Doh.